Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -4816,6 +4828,11 @@ namespace AzToolsFramework
             return false; // can't drop while tree is rebuilding itself!
         }
 
+        if (m_selectionContainsReadOnlyEntity)
+        {
+            return false;
+        }
+
```

Comment:
could this be apart of the last statement since it would be if either m_isAlreadyQueuedRefresh or m_selectionContainsReadOnlyEntity